src/fastoad/model_base/flight_point.py

```diff
@@ -12,16 +12,30 @@
 #  GNU General Public License for more details.
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
 from copy import deepcopy
-from dataclasses import asdict, dataclass, fields
+from dataclasses import asdict, dataclass, field, fields
 from numbers import Number
-from typing import Any, List, Mapping, Sequence, Union
+from typing import Any, Dict, List, Mapping, Optional, Sequence, Union
 
 import numpy as np
 import pandas as pd
 
 from fastoad.constants import EngineSetting
 
+FIELD_DESCRIPTOR = "field_descriptor"
+
+
+@dataclass
+class _FieldDescriptor:
+    """
+    Class to be used as dataclass field metadata.
+    """
+
+    is_cumulative: Optional[bool] = False
+    is_output: Optional[bool] = True
+    unit: Optional[str] = None
```

Comment:
Don't value and default value also need to be in the metadata ?

Reply:
Here, metadata are for handling data that are not handled by the dataclass. Value and default value are handled in the "classic" Python way.